lib/jekyll/readers/theme_assets_reader.rb

```diff
@@ -0,0 +1,47 @@
+module Jekyll
+  class ThemeAssetsReader
+    attr_reader :site
+    def initialize(site)
```

Comment:
This is copied from another reader. I don't have a personal preference and we don't have it enforced in our Rubocop config.